### PR TITLE
Refactored the first blog on kubeslice Office hours

### PR DESCRIPTION
**Pull request by 26tanishabanik**
Changed the published date
Changed the description and slug in the doc
Signed off by: Tanisha Banik [26tanishabanik@gmail.com](mailto:26tanishabanik@gmail.com)

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
@26tanishabanik please could you sign the DCO?

File: blog/2022-09-5-office-hours.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: KubeSlice Office Hours - 25th August 2022
-description: This is my first post on Docusaurus 2.
-slug: welcome-docusaurus-v2-one
+description: This is the first post of KubeSlice Office Hours.
```
Comment (divya-mohan0209) [inline blog/2022-09-5-office-hours.mdx:3]:
```suggestion
description: KubeSlice Office Hours, 25th August 2022.
```

File: blog/2022-09-5-office-hours.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+slug: KubeSlice Office Hours, 25th August 2022
```
Comment (divya-mohan0209) [inline blog/2022-09-5-office-hours.mdx:4]:
```suggestion
slug: 2022-08-25-kubeslice-office-hours
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Nits

**Comment by amolmote** [comment]
I think @26tanishabanik  you did signoff after opening the PR manually  if i am not wrong. 
Do one think for I have read the DCO action doc, for DCO it is mandatory to signoff while commiting changes to remote repo.
Still you can signoff while squashing all this 6 commits into one (usually it is called as rebase) getting my point??, if not then feel free to ask me on slack i will help you to do this.

**Pull request closed without merging**